go/noise/gaussian_noise_test.go

```diff
@@ -1058,14 +1110,14 @@ func TestComputeConfidenceIntervalFloat64Gaussian(t *testing.T) {
 	} {
 		got, err := gauss.ComputeConfidenceIntervalFloat64(tc.noisedX, tc.l0Sensitivity, tc.lInfSensitivity, tc.epsilon, tc.delta, tc.alpha)
 		if (err != nil) != tc.wantErr {
-			t.Errorf("ComputeConfidenceIntervalFloat64: when %s got err %v, wantErr=%t", tc.desc, err, tc.wantErr)
+			t.Errorf("ComputeConfidenceIntervalFloat64Gaussian: when %s got err %v, wantErr=%t", tc.desc, err, tc.wantErr)
 		}
 		if !approxEqual(got.LowerBound, tc.want.LowerBound) {
-			t.Errorf("TestComputeConfidenceIntervalFloat64(%f, %d, %f, %f, %f)=%0.10f, want %0.10f, desc %s, LowerBound is not equal",
+			t.Errorf("TestComputeConfidenceIntervalFloat64Gaussian(%f, %d, %f, %f, %f)=%0.10f, want %0.10f, desc %s, LowerBound is not equal",
```

Comment:
maybe "lower bounds are not equal"? (the same for upper)

Reply:
Changed the error message for other confidence interval tests too in gaussian_noise_test.go